Changes

```diff
@@ -609,6 +609,9 @@ Release branch for 4.06:
 - GPR#1073: Remove statically allocated compare stack.
   (Stephen Dolan)
 
+- GPR#1086: in Sys.getcwd, don't call getwd() if HAS_GETWD is not set
+  (report and first fix by Sebastian Markbåge, final fix by Xavier Leroy)
```

Comment:
This needs updating to reflect the fact that we no longer use `getwd` at all.

Reply:
Done.